### DISCUSSION TITLE
Consider adding "noindex" hint for search engine in aggregate pages (SEO purpose)

### DISCUSSION THREAD

**Discussion by wdhongtw** [ppoffice/hexo-theme-icarus#942] [Ideas]
Aggragate pages like `/tags/*` and `/categories/*` have aggragated contents from all pages under some tag / category.
It seems that it's hard for search engine to pick the canonical URL for some posts, bacause same content are presented in both original post page and the aggregated tag/category page.
(I may be wrong about this part.)

We can hint the search engin not to index the aggregate pages by adding `<meta name="robots" content="noindex">` in the header part.
The page will still be crawled (for discovering other posts), but will not included in indexed result
When the aggregate pages are not indexed, the original posts are more likely to be shown in top results.

BTW, `/archives` seems to have same issue, but I'm not sure whether it's a good to apply the same hint.

References:

- [Block Search Indexing with 'noindex' | Google Search Central](https://developers.google.com/search/docs/advanced/crawling/block-indexing)

**Comment by ppoffice** [comment]
Do you have a reference to this statement? I don't see why archive, category, and tag pages will prevent search engine from correctly index the posts.

> It seems that it's hard for search engine to pick the canonical URL for some posts, bacause same content are presented in both original post page and the aggregated tag/category page.

But the index pages do add a lot of search indices, which can be annoying for users.

PS:
See the latest commit: https://github.com/ppoffice/hexo-theme-icarus/commit/c8f9807d6ad2cdebb74d8fffc2ec1db500aaa003

**Reply by wdhongtw** [reply]
I can not provide a good reference to this statement, and it may be over-claimed...

When I search for some post titles of posts on my website, the tag page or the category page shows instead of the original post page.
That's the only part I can say objectively, and the rest statement is just a guess from my limited knowledge. T^T

Anyway, thanks for your immediate response and enhance of this project. :D
